Extensions/xkit_preferences.js

```diff
@@ -1006,7 +1006,7 @@ XKit.extensions.xkit_preferences = new Object({
 
 		if (obj.name.startsWith("xkit_") && XKit.tools.get_setting("xkit_show_internals","false") === "false") { return ""; }
 
-		var blacklisted_extensions = ["xkit_installer"];
+		var blacklisted_extensions = ["xkit_installer","old_stats"];
```

Comment:
I was under the impression that they were, but if they aren't then we
should notify people

On Sat, Nov 21, 2015 at 1:49 PM Wolvan <notifications@github.com> wrote:

> In Extensions/xkit_preferences.js
> <https://github.com/new-xkit/XKit/pull/739#discussion_r45549330>:
>
> > @@ -1006,7 +1006,7 @@ XKit.extensions.xkit_preferences = new Object({
> >
> >  		if (obj.name.startsWith("xkit_") && XKit.tools.get_setting("xkit_show_internals","false") === "false") { return ""; }
> >
> > -		var blacklisted_extensions = ["xkit_installer"];
> > +		var blacklisted_extensions = ["xkit_installer","old_stats"];
>
> is old_sidebar a full replacement to old_stats? I guess it's OK then. But
> I still think we should tell people since old_sidebar's default looks a lot
> different than old_stats. I bet that people would start to wonder what
> happened
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/new-xkit/XKit/pull/739/files#r45549330>.
>